azure-communication-ui/chat/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.Azurecommunicationui" parent="android:Theme.Material.Light.NoActionBar">
```

Comment:
For Calling theme name is "AzureCommunicationUICalling.Theme"
We should use same format for Chat composite as well